lib/components/NestedViews.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import {
+  Header,
+  Text,
+  Button,
+  Link,
+  utils
+} from '@bpanel/bpanel-ui';
+
+const Home = ({ location }) => (
+  <div>
+    <Header type="h3">Home View</Header>
+    <Text type="p">This is the "home view" that shows when at the parent path</Text>
+    <Text type ="p">Path: {location.pathname}</Text>
+  </div>
+);
+
+const NotFound = ({ location }) => (
+  <div>
+    <Header type="h3">404</Header>
+    <Text type="p">
+    You can optionally have a 404 view when you've navigated to an unregistered view.
```

Comment:
I wonder if escaping the `'` would help with the formatting on github

Reply:
unfortunately it didn't work. Just output a `\` :-/ . I could put it in brackets and then use template literals I suppose.